### PR TITLE
26 users can choose one of the black cards client

### PR DESCRIPTION
**Pull request by anhuwi**
The user can choose a blackcard. The black card is shown in the users profil page.
Problem with routing -> i opened new issue

### CODE REVIEW

File: src/components/views/PlayWhites.js [hunk 1]
```diff
@@ -4,52 +4,73 @@ import BaseContainer from "components/ui/BaseContainer";
...
+  const userId = localStorage.getItem("id")
+  const token = localStorage.getItem("token")
 
   // define a state variable (using the state hook).
   // if this variable changes, the component will re-render, but the variable will
   // keep its value throughout render cycles.
   // a component can have as many state variables as you like.
   // more information can be found under https://reactjs.org/docs/hooks-state.html
 
+    // fetch the blackCards from the server (it is the server's responsibility to give us 8 cards)
```
Comment (JoMuelle) [inline src/components/views/PlayWhites.js:25]: Black cards in comment, white cards in function?

File: src/components/views/PlayWhites.js [hunk 2]
```diff
@@ -4,52 +4,73 @@ import BaseContainer from "components/ui/BaseContainer";
...
+            }
+        }
+        fetchWhiteCards();
+    }, []);
+
+    // placeholder in case of failure
+    let content = <div>No Content Available</div>
+
+    if(cards) {
+        // TODO: onClick: ask to confirm this card, then make api call to submit this card choice, then reload page
```
Comment (JoMuelle) [inline src/components/views/PlayWhites.js:52]: See other PR, use window.confirm

File: src/components/views/PlayWhites.js (hunk 1 above)
Comment (anhuwi) [reply src/components/views/PlayWhites.js:25]: done (in other branch)

File: src/components/views/PlayWhites.js (hunk 2 above)
Comment (anhuwi) [reply src/components/views/PlayWhites.js:52]: done (in other branch)

**Pull request merged**